src/inmanta/server/services/compilerservice.py

```diff
@@ -276,7 +276,7 @@ async def run(self, force_update: Optional[bool] = False) -> Tuple[bool, Optiona
             result = await self._run_compile_stage("Recompiling configuration model", cmd, project_dir, env=env_vars_compile)
             success = result.returncode == 0
             if not success:
-                LOGGER.debug("Compile %s failed", self.request.id)
+                LOGGER.warning("Compile %s failed", self.request.id)
```

Comment:
The issue description seems to state that this should only be a warning for exporting compiles `self.request.do_export is True`. I'm not sufficiently familiar with the use case to be sure. @wouterdb you created the ticket, what was the intended behavior?

Reply:
Changed it in https://github.com/inmanta/inmanta/pull/2475/commits/1c98e58913d8f09979868d60e98213abdd540a66